### PR TITLE
fix background jobs

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
* Resolves: #3523 
* Target version: main

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] Sign-off message is added to all commits
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: lib/Backgroundjobs/Cleanup.php [hunk 1]
```diff
@@ -34,7 +34,8 @@ class Cleanup extends TimedJob {
 	/** @var WopiMapper $wopiMapper */
 	private $wopiMapper;
 
-	public function __construct(IDBConnection $db, WopiMapper $wopiMapper) {
+	public function __construct(ITimeFactory $time, IDBConnection $db, WopiMapper $wopiMapper) {
```
Comment (miaulalala) [inline lib/Backgroundjobs/Cleanup.php:37]: Missing the import

**Review by juliushaertl — APPROVED** [review]

**Comment by juliushaertl** [comment]
Cypress failure is unrelated

**Comment by juliushaertl** [comment]
Thanks ❤️

**Comment by kmorber** [comment]
I applied the fixes, but i still have the same problem.

**Pull request merged**